src/lib/Components/Bidding/Components/Title.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react"
+import { StyleSheet, TextProperties } from "react-native"
+
+import Serif from "../../Text/Serif"
+
+export class Title extends React.Component<TextProperties> {
+  render() {
+    const { children, style, ...props } = this.props
+
+    return (
+      <Serif style={[styles.default, style, styles.required]} {...props}>
+        {children}
+      </Serif>
+    )
+  }
+}
+
+const styles = StyleSheet.create({
```

Comment:
Came across things like this in the codebase. Not sure what the benefits are vs using `Serif`:
```
import styled from "styled-components/native"
import { Fonts } from "lib/data/fonts"

const Title = styled.Text`
  font-family: "${Fonts.GaramondRegular}";
  flex: 1;
  font-weight: bold;
  margin: 18px;
  text-align: center;
  font-size: 20;
`
```

Reply:
Also noticed that the font for the header is `AGaramondPro-Semibold` vs serif that has `AGaramondPro-Regular`. I guess we have to add that font because the `GaramondRegular` bold is thicker than semibold